src/modules/facility-investor/facility-investor.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { PROPERTIES } from '@ukef/constants';
+
+import { AcbsAuthenticationService } from '../acbs/acbs-authentication.service';
+import { AcbsFacilityPartyService } from '../acbs/acbs-facility-party.service';
+import { FacilityInvestorToCreate } from './facility-investor-to-create.interface';
+
+@Injectable()
+export class FacilityInvestorService {
+  constructor(private readonly acbsAuthenticationService: AcbsAuthenticationService, private readonly acbsFacilityPartyService: AcbsFacilityPartyService) {}
+
+  async createInvestorForFacility(facilityIdentifier: string, newFacilityInvestor: FacilityInvestorToCreate): Promise<void> {
+    const idToken = await this.acbsAuthenticationService.getIdToken();
+    const facilityPartyToCreateInAcbs = {
+      FacilityStatus: {
+        FacilityStatusCode: PROPERTIES.FACILITY_INVESTOR.DEFAULT.facilityStatus.facilityStatusCode,
+      },
+      InvolvedParty: {
+        PartyIdentifier: PROPERTIES.FACILITY_INVESTOR.DEFAULT.involvedParty.partyIdentifier,
+      },
+      EffectiveDate: newFacilityInvestor.effectiveDate + 'T00:00:00Z',
```

Comment:
Should we load `'T00:00:00Z'` this from constants as well since it has been repeated on multiple occasions?